deployment/wasm_pod_using_wde.yaml

```diff
@@ -0,0 +1,28 @@
+# Modified from krustlet hello world demo: https://github.com/deislabs/krustlet/blob/main/demos/wasi/hello-world-c/k8s.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: im-using-debug-echo-devices
+spec:
+  containers:
+    - name: im-using-debug-echo-devices
+      image: ghcr.io/rodz/wasi-hello-world:v1
+      resources:
+        requests:
+          "akri.sh/akri-debug-echo-43beb5" : 1
```

Comment:
Can we always expect the device to have this instance name: `akri.sh/akri-debug-echo-43beb5`? Will the hash change?

Reply:
Yes, this will always be present if following the demo tutorial as Akri debug echo configuration returns the foo0 object by default, so the hash will be the same.